gef-ui/server/utils/facility-amendments.helper.canUserAmendFacility.test.ts

```diff
@@ -0,0 +1,104 @@
+import * as dtfsCommon from '@ukef/dtfs2-common';
+import { DEAL_STATUS, DEAL_SUBMISSION_TYPE, ROLES } from '@ukef/dtfs2-common';
+import { canUserAmendFacility } from './facility-amendments.helper';
+import { Deal } from '../types/deal';
+import { Facility } from '../types/facility';
+
+const { AIN, MIN, MIA } = DEAL_SUBMISSION_TYPE;
+const { UKEF_ACKNOWLEDGED, DRAFT } = DEAL_STATUS;
+const { MAKER, CHECKER, ADMIN } = ROLES;
+
+const generateMockDeal = () =>
+  ({
+    submissionType: MIN,
+    status: UKEF_ACKNOWLEDGED,
+  }) as Deal;
+
+const generateIssuedFacility = () =>
+  ({
+    hasBeenIssued: true,
+  }) as Facility;
+
+describe('canUserAmendFacility', () => {
```

Comment:
Should we call this `userCan...` instead of `canUser...`? `userCan` is more common throughout the codebase